brightness.go

```diff
@@ -0,0 +1,88 @@
+package deskconnd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var BasePath = "/sys/class/backlight" //nolint: gochecknoglobals
+
+type Brightness struct {
+	brightnessFilePath string
+	maxBrightness      int
+	deviceExist        bool
+}
+
+func NewBrightness() *Brightness {
+	entries, err := os.ReadDir(BasePath)
+	if err != nil {
+		return &Brightness{deviceExist: false}
+	}
+
+	var device string
+	for _, e := range entries {
+		full := BasePath + "/" + e.Name()
+
+		info, err := os.Stat(full)
+		if err != nil {
+			continue
+		}
+
+		if info.IsDir() {
+			device = full
+			break
+		}
+	}
+
+	if device == "" {
+		return &Brightness{deviceExist: false}
+	}
+
+	b := &Brightness{
+		brightnessFilePath: device + "/brightness",
+		deviceExist:        true,
+	}
+
+	raw, err := os.ReadFile(device + "/max_brightness")
+	if err != nil {
+		return &Brightness{deviceExist: false}
+	}
+	b.maxBrightness, _ = strconv.Atoi(strings.TrimSpace(string(raw)))
+
+	return b
+}
+
+func (b *Brightness) GetBrightness() (int, error) {
+	if !b.deviceExist {
+		return 0, fmt.Errorf("brightness device not available")
+	}
+
+	raw, err := os.ReadFile(b.brightnessFilePath)
+	if err != nil {
+		return 0, err
+	}
+
+	current, _ := strconv.Atoi(strings.TrimSpace(string(raw)))
+	percent := (current * 100) / b.maxBrightness
+
+	return percent, nil
+}
+
+func (b *Brightness) SetBrightness(percent int) error {
+	if !b.deviceExist {
+		return fmt.Errorf("brightness device not available")
+	}
+
+	if percent < 0 {
+		percent = 0
```

Comment:
Please do not allow percent = 0.

On many laptop panels (including mine), writing 0 to the brightness file completely turns off the display, not just dims it.
I actually lost visibility of my entire screen during testing because of this.

To avoid this happening to other users, please enforce:

If the caller passes 0, internally treat it as 10%

Or at least enforce a safe minimum like 5–10%